cashctrl_ledger/ledger.py

```diff
@@ -19,6 +19,8 @@ class CashCtrlLedger(LedgerEngine):
     usage examples.
     """
 
+    _transitory_account =  None
```

Comment:
Too many spaces here: `=  None`

Reply:
I've pushed 2 commits:

- [bfad670](https://github.com/macxred/cashctrl_ledger/pull/63/commits/bfad6702e833aa195bac651ad32ed46df22b202b) Added missing docstrings for the methods and rebased branch onto the main
- [08e0988](https://github.com/macxred/cashctrl_ledger/pull/63/commits/08e0988b7a3ca220f763d1bdfea089e045c45e3c) Fixes revealed tests failure without setup `transitory_account`